<launch>
  <rosparam command="load" file="$(find assembly_part_recognition)/param/azure_scene.yaml" ns="azure_scene" />
  <node name="azure_scene_segmenter" pkg="assembly_part_recognition" type="azure_scene_segmenter.py" output="screen"/>
</launch>